_posts/2016-04-26-cloudready-linux-mint.markdown

```diff
@@ -0,0 +1,166 @@
+
+Neverware's focus is on revamping schools' old computer labs. Since a lot of that hardware is still better than today's Chromebooks, it runs Chromium beautifully. They also have a setup process to preserve the current Windows install and "dual boot" that with Cloudready.
+
+I use quotes around "dual-boot" because, when running the dual-boot setup, Cloudready is actually installed **into** the Windows partition, instead of along side it, completely taking it over. Its own boot software then manages whether it boots the user's Windows install or Chromium.
+
+I've provided diagrams below to explain the difference between this and a traditional dual-boot setup. We're aiming for a traditional dual-boot setup with Cloudready and Linux Mint in parallel. We may even try for the triple-boot!
+
+
+![Linux Mint and Cloudready boot structures](http://i.imgur.com/mK8mdFu.png)
+
+<center>*Source: Me, diagram baller*</center>
+
+##Chapter 1: Pre-install
+
+#####Get Cloudready bootable usb
+
+To "dual-boot" like this, the [Cloudready docs say](http://www.neverware.com/installation) you'll need 32GB of free space in the current Windows partition for Cloudready to use.
+
+Since my machine was already running Windows 7, I thought I'd try for a triple-boot. I first booted into the current Windows install to make sure I cleaned out all my old stuff off before starting anything.
+
+Within a few minutes Windows had already pissed me off, and I resolved that my gf wouldn't want to deal with Windows anyways. Consequently, I went with the Cloudready standalone install, also resolving to wipe everything from the machine and then install Mint after that.
+
+So with that in mind, follow the instructions [here](http://www.neverware.com/installation#quickinstallprocess) for getting the standalone version of Cloudready bootable from a usb stick that's at least 16GB.
+
+
+#####Get Linux Mint bootable usb
+
+Personally, I only had one usb stick available, but ideally you'd have another to put Mint on. I'll cover this later, but after the Cloudready install, this usb stick can be wiped and used for the Mint install.
+
+When ready, follow these [instructions to boot Mint from a usb](https://community.linuxmint.com/tutorial/view/744).
+
+
+#####Setting up the BIOS
+
+Throughout the setup process you may come across a few recommended changes to the BIOS to get everything working. Might as well do these now. This HP machine has its own HP version of the BIOS, so some things may be specific to HP. I tried to keep things as general as possible, but the [archlinux wiki](https://wiki.archlinux.org/index.php/GRUB/EFI_examples) has detailed info for setting the UEFI mode for any BIOS.
```

Comment:
Shoutouts to arch linux for having such good docs that other distros don't even need their own